src/lib/mina_base/test/zkapp_command/fee_related.ml

```diff
@@ -0,0 +1,37 @@
+open Core_kernel
+open Mina_base.Zkapp_command
+open Mina_base
+open Snarky_backendless
+open Snark_params.Tick
+
+let%test_module "fee_related_tests" =
+  ( module struct
+    let feepayer_body_generator = Zkapp_command.T.Stable.Latest.Wire.gen
+
+    let%test_unit "fee" =
+      Quickcheck.test ~trials:50 feepayer_body_generator ~f:(fun x ->
+          [%test_eq: Currency.Fee.t] x.fee_payer.body.fee
+            (fee @@ Zkapp_command.of_wire x) )
```

Comment:
`Zkapp_command` is already open. However, given that most functions called from this module are from there, I'd prefer to only open it locally. This arrangement gives the impression that `fee` and `of_wire` functions come from different modules, which is not the case. In short, I suggest to rather write: `Zkapp_command.(fee @@ of_wire x)` and remove the `open` at the top of the module.